src/hooks/client.ts

```diff
@@ -97,11 +97,13 @@ export function useSetOwnDeviceInfo() {
 	const queryClient = useQueryClient()
 	const clientApi = useClientApi()
 
-	const { mutate, status, reset } = useMutation(
+	const { error, mutate, reset, status } = useMutation(
 		setOwnDeviceInfoMutationOptions({ clientApi, queryClient }),
 	)
 
-	return { mutate, reset, status }
+	return status === 'error'
+		? { error, mutate, reset, status }
+		: { error: null, mutate, reset, status }
```

Comment:
in theory this also would've worked 😅 

```ts
return status === 'error'
		? { error, mutate, reset, status }
		: { error, mutate, reset, status }
```

Open to better ways of getting this to work as intended!